src/debugger/nodeDebugWrapper.ts

```diff
@@ -132,6 +133,7 @@ Telemetry.init("react-native-debug-adapter", version, true).then(() => {
         args.args = [
             args.platform,
             debugServerListeningPort.toString(),
+            args.internalExtensionPort.toString(),

```

Comment:
I would suggest also adding in default-setting logic here to avoid any issues with unexpected input since it is user-exposed. That would also prepare it for when we move the bulk of the initialisation code to the debug adapter and so the value would be consumed here as well.